src/features/chart/components/Chart.tsx

```diff
@@ -25,9 +26,9 @@ const tenors = [
   { value: "max", label: "MAX" }
 ];
 
-const yaxisFormat = item => item.toFixed(2);
+const yaxisFormat = (item: any) => item.toFixed(2);
 
-const formatDate = (isoDate, ChartRange) => {
+const formatDate = (isoDate: any, ChartRange: any) => {
```

Comment:
Same applies here. You should know what `isoDate` and `ChartRange` actually are, and don't need to `any` type it.